Mathlib/RingTheory/KrullDimension/Basic.lean

```diff
@@ -101,10 +101,12 @@ lemma Ideal.mem_minimalPrimes_iff_isPrime [Ring.KrullDimLE 0 R] {I : Ideal R} :
     I ∈ minimalPrimes R ↔ I.IsPrime :=
   ⟨(·.1.1), fun _ ↦ I.mem_minimalPrimes_of_krullDimLE_zero⟩
 
+theorem nilradical_le_jacobson (R) [CommRing R] : nilradical R ≤ Ring.jacobson R :=
+  nilradical_eq_sInf R ▸ le_sInf fun _I hI ↦ sInf_le (Ideal.IsMaximal.isPrime ⟨hI⟩)
+
 theorem Ring.jacobson_eq_nilradical_of_krullDimLE_zero (R) [CommRing R] [KrullDimLE 0 R] :
     jacobson R = nilradical R := by
-  refine nilradical_eq_sInf R ▸ (le_sInf fun I hI ↦ sInf_le ?_).antisymm
-    (le_sInf fun _I hI ↦ sInf_le <| Ideal.IsMaximal.isPrime ⟨hI⟩)
+  refine (nilradical_le_jacobson R).antisymm' (nilradical_eq_sInf R ▸ le_sInf fun I hI ↦ sInf_le ?_)
```

Comment:
Since you are at it, can you replace the `change` line below with `rw [Set.mem_def, Set.setOf_app_iff] at hI`? Thanks.